dashboards/grafana-dashboard-kessel-relations-api-data-sync.configmap.yaml

```diff
@@ -519,7 +470,7 @@ data:
         "list": [
           {
             "current": {
-              "selected": true,
+              "selected": false,
```

Comment:
I'm not 100% on this, but I believe this selected value if set to true will ensure the datasource is set to crcs02ue1-prometheus on load but can't find info to correlate that.

Not required for merge but pointing it out in case im correct and you want to change that later

Reply:
Changed.